Sources/Hummingbird/Router/BinaryTrie/BinaryRouterResponder.swift

```diff
@@ -12,23 +12,20 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// Directs requests to handlers based on the request uri and method.
-///
-/// Conforms to `Responder` so need to provide its own implementation of
-/// `func respond(to request: Request, context: Context) async throws -> Response`.
-///
-public struct RouterResponder<Context: BaseRequestContext>: HTTPResponder {
-    let trie: RouterPathTrie<EndpointResponders<Context>>
+import NIOCore
+
+public struct BinaryRouterResponder<Context: BaseRequestContext>: HTTPResponder {
```

Comment:
I think we can now just rename it back then, since "Binary" is the new default

Reply:
```suggestion
public struct RouterResponder<Context: BaseRequestContext>: HTTPResponder {
```

Reply:
done and move file back into Router folder